driver-mapping/src/test/java/com/datastax/driver/mapping/MapperKeyspaceOptionTest.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (C) 2012-2017 DataStax Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.driver.mapping;
+
+import com.datastax.driver.core.CCMTestsSupport;
+import com.datastax.driver.core.Row;
+import com.datastax.driver.core.TestUtils;
+import com.datastax.driver.mapping.annotations.PartitionKey;
+import com.datastax.driver.mapping.annotations.Table;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static com.datastax.driver.core.Assertions.assertThat;
+
+public class MapperKeyspaceOptionTest extends CCMTestsSupport {
+
+    private MappingManager mappingManager;
+    private Mapper<User> mapper;
+    private String otherKeyspace;
+
+    @Override
+    public void onTestContextInitialized() {
+        otherKeyspace = TestUtils.generateIdentifier("ks_");
+        execute("CREATE TABLE user (key int primary key, v text)",
+                String.format("CREATE KEYSPACE IF NOT EXISTS %s WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1}", otherKeyspace),
```

Comment:
`IF NOT EXISTS` requires 2.0, so would be good to annotate the class with `CassandraVersion(major="2.0")`